### PR TITLE
Simple Teliop

### PR DESCRIPTION
**Pull request by zoecarver**
refs #8

### CODE REVIEW

File: TeamCode/src/main/java/org/redshiftrobotics/lib/RobotHardware.kt [hunk 1]
```diff
@@ -24,4 +27,30 @@ class RobotHardware(val ctx: LinearOpMode) {
             else -> PIDCalculator.Tuning.EMPTY
         }
     }
+
+    private var _leftPower: Float = 0.0f
```
Comment (ariporad) [inline TeamCode/src/main/java/org/redshiftrobotics/lib/RobotHardware.kt]: Can we make this a `Double`? I see no reason for any doubles to be anywhere.

File: TeamCode/src/main/java/org/redshiftrobotics/lib/RobotHardware.kt [hunk 2]
```diff
@@ -24,4 +27,30 @@ class RobotHardware(val ctx: LinearOpMode) {
...
+            return _leftPower
+        }
+        set(value) {
+            this._leftPower = value
+
+            this.leftFrontDrive.power = -value.toDouble()
+            this.leftBackDrive.power = -value.toDouble()
+        }
+
+    private var _rightPower: Float = 0.0f
```
Comment (ariporad) [inline TeamCode/src/main/java/org/redshiftrobotics/lib/RobotHardware.kt]: Ditto

File: TeamCode/src/main/java/org/redshiftrobotics/lib/RobotHardware.kt (hunk 1 above)
Comment (ariporad) [inline TeamCode/src/main/java/org/redshiftrobotics/lib/RobotHardware.kt]: Also, we should just use a `field` for these: https://kotlinlang.org/docs/reference/properties.html#backing-fields

File: TeamCode/src/main/java/org/redshiftrobotics/opmodes/Teliop.kt [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package org.redshiftrobotics.opmodes
+
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp
+import org.redshiftrobotics.lib.RobotHardware
+
+@TeleOp(name = "Test Teliop")
```
Comment (ariporad) [inline TeamCode/src/main/java/org/redshiftrobotics/opmodes/Teliop.kt]: TeleOp is spelled with is spelled with two `e`s and no `i`s.

File: TeamCode/src/main/java/org/redshiftrobotics/lib/RobotHardware.kt (hunk 1 above)
Comment (michaelconnolly) [reply TeamCode/src/main/java/org/redshiftrobotics/lib/RobotHardware.kt]: agreed.

**Comment by zoecarver** [comment]
@ariporad I think you said resolved this yesterday. If not that's fine, I can do it (just let me know).

**Comment by ariporad** [comment]
Closing this in favor of #19.

**Pull request closed without merging**